datasets/esa-worldcover/dataset.yaml

```diff
@@ -0,0 +1,28 @@
+id: esa-worldcover
+image: ${{ args.registry }}/pctasks-esa-worldcover:latest
+
+args:
+  - registry
+
+code:
+  src: ${{ local.path(./esa_worldcover.py) }}
+
+environment:
+  AZURE_TENANT_ID: ${{ secrets.task-tenant-id }}
+  AZURE_CLIENT_ID: ${{ secrets.task-client-id }}
+  AZURE_CLIENT_SECRET: ${{ secrets.task-client-secret }}
+
+collections:
+  - id: esa-worldcover
+    template: ${{ local.path(./collection) }}
+    class: esa_worldcover:ESAWorldCoverCollection
+    asset_storage:
+      - uri: blob://ai4edataeuwest/esa-worldcover/
+        chunks:
+          options:
+            extensions: [.tif]
+            name_starts_with: v200/2021/map
```

Comment:
Just checking that we're only supporting ingest of for the 2021 data?

Reply:
No, but it seemed reasonable to split by year since we may treat the 2020 update differently than the 2021 ingest. And because it will make it easier to ingest only 2022 data when that becomes available. I added a comment to this effect in the `dataset.yaml`, and also created a workflow yaml for each year and placed them in a "workflows" directory.